internal/ingress/controller/config/config.go

```diff
@@ -492,6 +496,7 @@ func NewDefault() Configuration {
 	cfg := Configuration{
 		AllowBackendServerHeader:   false,
 		AccessLogPath:              "/var/log/nginx/access.log",
+		WorkerCpuAffinity:          "",
```

Comment:
Use `auto` as default. This way you don't need a condition in the template

Reply:
I have thought about this. However the default nginx behavior is not bind process to any cpus. And if `auto` is the default, the condition is still needed if user don't want any binding

Reply:
ok, makes sense